tests/introspection/ApiCoreImageFiltersTest.cs

```diff
@@ -54,7 +54,7 @@ public abstract class ApiCoreImageFiltersTest : ApiBaseTest {
 
 		static Type CIFilterType = typeof (CIFilter);
 
-#if false
+#if true
```

Comment:
@spouliot looking from mobile but was this an intended change?

Reply:
yes, it was spitting out bindings (if missing) in logs
no a huge deal since none should be missing... but :)